src/Http/Http.Abstractions/src/Microsoft.AspNetCore.Http.Abstractions.WarningSuppressions.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<linker>
```

Comment:
All of these are effectively a bug.

Suppressions are dangerous and should fall into two buckets:
* Something which is truly not problematic and we've spent the effort to prove that it's not a problem. In this case it should be suppressed directly in the code using the `UnconditionalSuppressMessageAttribute` on the affected method - with a justification/comment of why we thing it's OK. These should be rare - most likely just workarounds for limitations in the trimmer.
* Special cases, where we want the warning to be shown to the user and we can't fully annotate an API - this is done in runtime in only a handfull of places and it's typically around special features which are partially handled by native code (and so it's invisible to the trimmer). This should be SUPER rare and in almost all cases hidden behind a feature switch.

Runtime uses the XML attributes to suppress warnings in two scenarios:
* Warnings which we didn't get to fixing yet - so they are truly bugs, but we wanted to baseline the warnings for the build.
* Warnings which we want to keep in the code (as mentioned above) intentionally (very rare) - again we want to baseline these for the build.

Notice that in all of these cases the suppressions are NOT present in the final assembly used by developers - they're only passed to the trimmer on a command line - to effectively baseline (Selectively suppress) warnings during our own builds.